rust_snuba/src/processors/replays.rs

```diff
@@ -184,15 +185,23 @@ pub fn deserialize_message(
             }]
         }
         ReplayPayload::EventLinkEvent(event) => {
-            let level_id = event
+            let result = event
                 .debug_id
                 .or(event.error_id)
                 .or(event.fatal_id)
                 .or(event.info_id)
                 .or(event.warning_id)
-                .unwrap_or_default();
-            if level_id.is_nil() {
-                return Err(anyhow!("missing level id"));
+                .ok_or(anyhow!("missing level id"));
+            if result.is_err() {
+                add_breadcrumb(Breadcrumb {
```

Comment:
oh i think you can just do `log::debug!` or something similar, it should be caught as breadcrumb.